components/LinkCard.jsx

```diff
@@ -0,0 +1,15 @@
+import styles from "@/styles/components/link-card.module.scss";
+
+export default function LinkCard({ name, url, icon }) {
+	return (
+		<a className={styles["link-card"]} href={`#${url}`}>
```

Comment:
wrote it in a previous comment, I missed it. make the href usable for future cases. e.g: ```href={`/${url}`}```

Reply:
noted